components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/utils/AuthUtil.java

```diff
@@ -131,11 +132,16 @@ public static String extractTokenFromHeaders(Headers headers, String cookieHeade
         } else {
             return null;
         }
+
+        //Append unique environment name in deployment.yaml
+        String environmentName = ConfigurationService.getEnviornmentName();
```

Comment:
Spelling error in method name "getEnviornmentName()". Shall we correct it to "getEnvironmentName()"

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/4702/commits/7c3c6206a6826c02f027cc5c99e7cfdb4608d36d